dashboard-frontend/src/types/study.ts

```diff
@@ -1,31 +1,36 @@
+import { Currency } from './shared';
+
 export interface Study {
   id: string;
   name: string;
   slug: string;
-  createdAt: string;
+  createdAt: number;
   studyProgressList: StudyProgress[];
   stratumProgressList: StratumProgress[];
 }
 
-interface StudyProgress {
+export interface StudyProgress {
   id: string;
-  datetime: string;
+  datetime: number;
+  desiredParticipants: number;
```

Comment:
I think this one was missing here. In StratumProgress we have it.

Reply:
Interesting. Yeah let's have it. It won't be there for every study, nor will it be in StratumProgress always (often its percentage), but it can be in absolute terms so let's keep it!

Reply:
Perfect, I'll keep it then 👍

@nandanrao  Do we have the percentage always ? or it is also missing sometimes ?

If it's always there we can use it to always compute desiredParticipants at the api level, but if it's not always there I need to update the types to take into account this.

Reply:
Ok - sorry I wasn't clear on this before: 

 `desiredParticipants?` should be optional in both the types, on study level and stratum level!

The rest we should always have. 

One option is to tell us how many they want, another is to say "just get as many as you can, given this perentage balance".

Reply:
Oooh this last paragraph is key, now everything is clear 👌

I'll make both of them optional. Thanks!!